client/views/curatorInbox/curatorSourceDetails.jade

```diff
@@ -53,9 +53,20 @@ template(name="curatorSourceDetails")
               source=source)
           if notifying
             .veil
+        else if enhancements.error
+          .curator-inbox--warning
+            i.fa.fa-warning
+            h2=enhancements.error
+            .btn.retry Retry
+        else if enhancements.processingStartedAt
```

Comment:
Yes, sometimes the document will get stuck in the processing state and this makes it possible to retry it.